tests/gameplay.test.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-undef */
+import Gameboard from '../src/modules/data/gameboard';
+import Player from '../src/modules/data/players';
+
+test('receives attack and updates target position', () => {
+  const playerBoard = Gameboard();
+  const computerBoard = Gameboard();
+  const player = Player('player', playerBoard, playerBoard.data);
+  player.attack(computerBoard, 0, computerBoard.data);
+  expect(computerBoard.data[0]).not.toBe(null);
+});
```

Comment:
**[ REQUIRED ]** This change should be part of the gameboard test suite, however you are asking for the cell not to have the porperty of null, when you are testing if an attack was recieved you could check for the string(in this case X or O) to be in that particular cell. This should also check for hits or miss.